docker/auth/auth.py

```diff
@@ -168,11 +167,17 @@ def load_config(config_path=None):
     try:
         with open(config_file) as f:
             data = json.load(f)
+            res = {}
             if data.get('auths'):
                 log.debug("Found 'auths' section")
-                return parse_auth(data['auths'])
+                res.update(parse_auth(data['auths']))
+            if data.get('HttpHeaders'):
+                log.debug("Found 'HttpHeaders' section")
+                res.update({'HttpHeaders': data['HttpHeaders']})
+            if res:
+                return res
```

Comment:
I think this could use a new test case. I don't see anything asserting these are read properly.